src/index.js

```diff
@@ -55,6 +55,12 @@ import createHeatMapTests from './project-tests/heat-map-tests';
 export const assert = chai.assert;
 
 let projectNameLocal = false;
+let localStorageByProject = function() {
+  const placeholder = document.getElementById('placeholder');
+  const visibleProjectName = placeholder.innerHTML.split(' ').join('');
+  console.log(visibleProjectName);
```

Comment:
Oops, you should remove this `console.log`. Sorry if I missed this the first time around.